crates/cadenza-eval/STATUS.md

```diff
@@ -85,9 +85,14 @@ The evaluator implements a minimal tree-walk interpreter for Cadenza. It can:
    - [x] Exported `Type` from crate root
    - [PR Comment](https://github.com/camshaft/cadenza/pull/4#discussion_r2573079828)
 
-6. **Values need syntax nodes for source tracking**
-   - Current: Values have no source location info
-   - Needed: Attach syntax nodes, handle multi-file tracking, add Expr as runtime value
```

Comment:
I'm not totally sure we're done with this item, as is. And maybe I should have been more specific about what wer were trying to accomplish. Basically the idea is that we want to provide good diagnostics when errors happen. But the more I think about it, it's possible that it really only applies to identifiers. More specifically, I'd like to be able to say something like "tried to call this function defined in file:line with the wrong number of arguments" or "tried to call the variable `foo` which was defined at file:line but the type was an Integer". So maybe we don't need something generic like we've got so far in this PR. We more need to attach locations to variables/scopes and function definitions.